themes/godotengine/layouts/download.htm

```diff
@@ -123,11 +123,12 @@
     <div class="main">
       <h1 class="intro-title">Download</h1>
     </div>
-    <div class="tabs">
+    <div class="tabs tabs-download">
       <a href="/download/linux" class="title-font {% if platform == 'linux' %} active {% endif %}">Linux</a>
       <a href="/download/osx" class="title-font {% if platform == 'osx' %} active {% endif %}">macOS</a>
       <a href="/download/windows" class="title-font {% if platform == 'windows' %} active {% endif %}">Windows</a>
       <a href="/download/server" class="title-font {% if platform == 'server' %} active {% endif %}">Linux Server</a>
+      <a href="/download/android" class="title-font {% if platform == 'windows' %} active {% endif %}">Android</a>
```

Comment:
```suggestion
      <a href="/download/android" class="title-font {% if platform == 'android' %} active {% endif %}">Android</a>
```
Provided that it works? Otherwise it shouldn't be Windows I guess.